packages/smooth_app/lib/pages/product/nutrition_page/nutrition_page.dart

```diff
@@ -249,6 +250,41 @@ class _NutritionPageLoadedState extends State<NutritionPageLoaded>
       }
     }
 
+    final NutritionValidator validator = NutritionValidator();
+
+    String servingSize = '100.0';
+
+    if (_nutritionContainer.perSize == PerSize.serving) {
+      servingSize = _nutritionContainer.servingSize;
+    }
```

Comment:
Not sure what you want to do here, but a cleaner syntax would be:
```suggestion
    final String servingSize;
    if (_nutritionContainer.perSize == PerSize.serving) {
      servingSize = _nutritionContainer.servingSize;
    } else {
      servingSize = '100.0';
    }
```

Reply:
The scope is to obtain the serving size weight. There are 2 options:  per serving and per 100 g/ml

![Screenshot 2025-04-27 191707](https://github.com/user-attachments/assets/baa24536-77d9-487f-95bb-227eee30ee88)

![Screenshot 2025-04-27 191722](https://github.com/user-attachments/assets/b9c553ce-f27a-41f5-b8c4-bece3ad8d4f9)

In the first case the weight is specified in the serving size field (and the user can specify the unit).